packages/widget/src/hooks/use-assets.ts

```diff
@@ -16,7 +18,7 @@ export function useAssets(options: AssetsRequest = {}) {
         includeCW20Assets: true,
         includeSvmAssets: true,
         ...options,
-      });
+      }) as Promise<Record<string, Asset[]>>;
```

Comment:
this works, but is bad. the proper way to do this would be to make sure `skipClient.assets` returns the correct type